src/screens/WhotoFollow/index.js

```diff
@@ -30,7 +30,7 @@ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {Header} from '../../components';
 import {InitialStartupAtom} from '../../service/initialStartup';
 import {ProgressBar} from '../../components/ProgressBar';
-import {normalizeFontSize} from '../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `fonts` from the `../../utils/fonts` module. This change is likely made to use the new font styles in the component. Ensure that the `fonts` object is correctly exported from the `../../utils/fonts` module and contains the necessary properties.
```diff
- import {normalizeFontSize} from '../../utils/fonts';
+ import {fonts, normalizeFontSize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->